gooey/gui/application.py

```diff
@@ -24,6 +24,7 @@ def run(build_spec):
 
 def build_app(build_spec):
   app = wx.App(False)
+  app.SetAppDisplayName(build_spec['program_name'])

```

Comment:
Could you add a comment here explaining that this for getting MacOS to display things correctly? 

Without it, a year from now, this'd probably be something I accidentally delete while assuming it was redundant or something.

Reply:
Done